doc.html

```diff
@@ -16,1281 +16,982 @@
     <div class="pf-c-page">
         <header role="banner" class="pf-c-page__header header">
             <div class="pf-c-page__header-brand" style="position: absolute;">
-               <a class="pf-c-page__header-brand-link" href="index.html">
-               <img class="pf-c-brand" src="./images/pbench_logo.png" alt="PatternFly logo" id="navlogo"/>
-               </a>
+                <a class="pf-c-page__header-brand-link" href="index.html">
+                    <img class="pf-c-brand" src="./images/pbench_logo.png" alt="PatternFly logo" id="navlogo" />
+                </a>
```

Comment:
Without necessarily objecting to the reformatting ... it would have been a lot easier to review if the reformatting had been in a separate commit separated from "real" changes.

Reply:
@dbutenhof the changes are mostly about the layout of the page. I kind of wrapped the elements inside others which was hectic to do without reformatting. But I will make sure to add them as separate commits from next time.